silnlp/common/extract_corpora.py

```diff
@@ -26,6 +29,10 @@ def main() -> None:
 
     projects: Set[str] = set(args.projects)
 
+    task = Task.init(
```

Comment:
I guess this is mostly done as a test ClearML task. I don't know how useful this would be in real-world use.